src/univers/nuget.py

```diff
@@ -0,0 +1,163 @@
+#
+# Copyright (c) nexB Inc. and others.
```

Comment:
Is this rather copyright Google? You probably want an initial commit with the original unmodified files? And did you modify this code?
I see that https://github.com/google/osv/blob/f5647ad2f746685b08debfba0293e442f2fb9945/lib/osv/semver_index.py#L1 and https://github.com/google/osv/blob/f5647ad2f746685b08debfba0293e442f2fb9945/lib/osv/nuget.py both have a notice and we surely do not want to remove their copyrights, even if this is still in the ABOUT and NOTICE file